packages/@glimmer/runtime/test/initial-render-test.ts

```diff
@@ -235,16 +232,16 @@ class Rehydration extends AbstractRehydrationTests {
     );
 
     env = this.delegate.clientEnv;
-    let clientRemote = (remote = env.getDOM().createElement('remote') as HTMLElement);
-    let host = env.getDOM().createElement('div') as HTMLElement;
+    let clientRemote = (remote = env.getDOM().createElement('remote'));
+    let host = env.getDOM().createElement('div');
     host.appendChild(this.element);
     host.appendChild(clientRemote);
-    clientRemote.innerHTML = serializedRemote;
-    this.element = host.firstChild as HTMLElement;
+    (clientRemote as Element).innerHTML = serializedRemote;
```

Comment:
can we hide this in a TestHost interface? that provides this op and whether it is supported by the underlying DOM? it would be nice to get to the point we can run the whole suite in SSR vs not for most tests.

Reply:
I tried untangling this one but it was quite hairy and I was afraid of making substantive changes in a PR this big. Will follow up with you offline because I agree we should clean this up.